### DISCUSSION TITLE
How to set the initial width of the sidebar?

### DISCUSSION THREAD

**Discussion by skwaruch** [piroor/treestyletab#3636] [Q&A]
In the extension options there is a sidebarwidthinwindow parameter, which is responsible for the sidebar width. When the browser starts it has a value of 420, how do I change this value?

**Comment by irvinm** [comment]
I didn't review the code too closely, but sidebarwidthinwindow I believe is just holding the sidebar width ... not setting or adjusting it.

To set the sidebar width check this:  https://github.com/piroor/treestyletab/discussions/3533
